### ISSUE TITLE
`bento init` fails installing peer deps for eslint

### ISSUE THREAD

**Issue by pabloest** [returntocorp/bento#10]
I cloned the [Netlify CMS project](https://github.com/netlify/netlify-cms) and tried to initialize Bento there. It failed with the following message:
```

install-peerdeps v1.10.2
It seems as if you are using Yarn. Would you like to use Yarn for the installation? (y/n) y
Installing peerdeps for eslint-config-airbnb@latest.
yarn add eslint-config-airbnb@18.0.1 eslint@6.1.0 eslint-plugin-import@^2.18.2 eslint-plugin-jsx-a11y@^6.2.3 eslint-plugin-react@^7.14.3 eslint-plugin-react-hooks@^1.7.0 --dev

yarn add v1.17.3
error Running this command will add the dependency to the workspace root rather than the workspace itself, which might not be what you want - if you really meant it, make it explicit by running this command again with the -W flag (or --ignore-workspace-root-check).
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
ERR The install process exited with error code 1.
Could not execute "npx install-peerdeps --dev eslint-config-airbnb":
None
```

Then I ran `bento init` again and Bento seemed to initialize successfully.

**Comment by nbrahms** [comment]
Running
```
npx install-peerdeps --dev eslint-config-airbnb
```
manually solves this issue, so long as yarn is _not_ used for install.

**Comment by nbrahms** [comment]
Fixed in 0.1.0r1
